maven-artifact/src/main/java/org/apache/maven/artifact/DefaultArtifact.java

```diff
@@ -207,6 +208,16 @@ public File getFile()
         return file;
     }
 
+    public Path getPath()
+    {
+        return this.getFile() == null ? null : this.getFile().toPath();
+    }
+
+    public void setPath( Path path )
+    {
+        this.setFile ( path == null ? null : path.toFile() );
+    }
+
```

Comment:
did you override default implementation from interface?

Reply:
I do not understand the question (or why you are asking it). Actually that above code *is* effectively overriding the default implementation of the interface, and it was explicitly done because @rmannibucau asked for this.

Reply:
I asking because body of method in default method in interface is the same as overridden one.
So maybe body in interface is redundant for first look, and I ask.
Ok, I have read previous comments - I didn't know about proxy problems.